Created nationalities table and added a nationality_id  column to the…

Mawadddah opened the pull request:
… profiles table with a foreign key to a nationality as a value

Mawadddah committed:
Created nationalities table and added a nationality_id  column to the profiles table with a foreign key to a nationality as a value

tekul commented on server/db/populate-db.sql:
```diff
@@ -10,3 +10,8 @@ INSERT INTO profiles (first_name, last_name, gender, date_of_birth, email, addre
 INSERT INTO profiles (first_name, last_name, gender, date_of_birth, email, address, phone_number, occupation) values ('Mimi', 'Mouse', 'female', '1967-01-30', 'happy_mood@cyf.org', '176 Battal Gazi sok., Dalaman', '+44 1345 56788', 'teacher');
+
+COPY nationalities(nationality)
+FROM '/home/coder/codeyourfuture/scot-min-imece/nationalities.csv'
```
Can you check if this works with a relative path? You can't use a hard-coded full path like `/home/coder/codeyourfuture/scot-min-imece/nationalities.csv` since that won't necessarily be valid on other machines.

Also if you use this (and it works), you need to add the CSV file to git too. I would put it in the `server/db` directory since that is the only place it will be used.

tekul commented on server/db/recreate-schema.sql:
```diff
@@ -19,8 +19,14 @@ CREATE TABLE profiles (
+
+CREATE TABLE nationalities (
+  id SERIAL PRIMARY KEY,
+ nationality   VARCHAR(100) NOT NULL
```
Make sure the indentation is correct :slightly_smiling_face: .

tekul replied:
Also the nationalities table needs to be defined before the profiles table since the latter needs it.

Mawadddah committed:
Fix nationalities sql

- Changed copy to insert for simplicity
- Fix order of the table drop and creation statements

tekul commented:
Merged manually
